src/stores/MessageStore.js

```diff
@@ -86,7 +86,10 @@ class MessageStore extends DataStore {
   async _fetchMany(options = {}) {
     const data = await this.client.api.channels[this.channel.id].messages.get({ query: options });
     const messages = new Collection();
-    for (const message of data) messages.set(message.id, this.add(message));
+    
+    const parsed = data.reverse().map(m => this.add(m))
+    for (const message of parsed.reverse()) messages.set(message.id, message)
```

Comment:
I dislike the whole reverse reverse again, as it does useless reversing. 

You could do something like ```js
const reversed = data.reverse();
for (const message of reversed) messages.set(message.id, this.add(message))
```

Although that's just a suggestion, and others might have another idea

Reply:
`for (const message of data.reverse()) messages.set(message.id, this.add(message));` even

also, you are missing semi-colons, and have some trailing spaces

Reply:
> you are missing semi-colons, and have some trailing spaces

I'd just setup prettier, no point wasting time on formatting.

Decided to just to create an internal fork of discord.js for my company's unique use case, gonna close.

Thanks